var/spack/repos/builtin/packages/pdal/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Pdal(CMakePackage):
+    """PDAL is a C++ library for translating and manipulating point cloud data.
+    It is very much like the GDAL library which handles raster and vector data.
+    """
+
+    homepage = "https://pdal.io"
+    url = "https://github.com/PDAL/PDAL/archive/refs/tags/2.6.2.tar.gz"
+
+    license("BSD Open Source license.")
```

Comment:
```suggestion
    license("BSD")
```

Sorry I didn't catch this earlier, licenses should be specified in SPDX format as per the [docs](https://spack.readthedocs.io/en/latest/packaging_guide.html#specifying-license-information).

Reply:
I googled it. it seems there multiple BSD versions. So, my understanding is that BSD open source license is different from the original BSD license.

Reply:
![image](https://github.com/spack/spack/assets/2924561/449940d2-fe1c-432c-a4fc-7a0054442262)

Reply:
let's get help from the developers of PDAL.

Reply:
https://github.com/PDAL/PDAL/blob/master/LICENSE.txt

Reply:
I'll approve the commit just in case.

Reply:
I'm told in the linked issue that the precise license is BSD-3-Clause-Attribution.html